Core/source/core/pgp-key.ts

```diff
@@ -182,7 +191,7 @@ export class PgpKey {
     await encryptKey({ privateKey: (prv as PrivateKey), passphrase });
   }
 
-  public static normalize = async (armored: string): Promise<{ normalized: string, keys: Key[] }> => {
+  public static normalize = async (armored: string): Promise<{ normalized: string, keys: Key[], exception: string }> => {
```

Comment:
should be optional, i.e. `exception?: string`, i.e. it can be undefined.

Reply:
name `exception` is a bit misleading. It could be `exception` if you would return here the whole exception object, but only error message is returned, as I can see later in the code. So I suggest to rename it into `error`, (and moreover, this will be more aligned with what we already have in this codebase).

Reply:
oh, okay sir. I've rename it appropriately to `error` field, making it optional and removing the error from the returning data. thanks!